doc/docusaurus/docs/reference/CIP-0069-changes.md

```diff
@@ -0,0 +1,177 @@
+---
+sidebar_position: 26
+---
+
+# Understanding CIP-0069 changes
+
+## Overview
```

Comment:
We should mention that CIP-69 and CIP-117 only applies for the upcoming PlutusV3. Existing PlutusV1 and PlutusV2 scripts will continue to work unchanged.

This document concerns those that want to write scripts in PlutusV3 or those that want to migrate their PlutusV1 & PlutusV2 scripts to PlutusV3.